src/types/fetchers.ts

```diff
@@ -14,13 +14,16 @@ export type CryptoCompareHistoFetcherResult = [
 
 export type CryptoCompareMultiProcessorResult = number | undefined;
 
+export type ByBitMultiProcessorResult = number | undefined;
```

Comment:
maybe we can unify with `CryptoCompareMultiProcessorResult`? or simply ignore if the type is the same?

Reply:
I created the `NumberProcessorResult` to unify them.
The problem of ignoring is to import in the fetcher. I can just use `number | undefined` instead
I don't have a strong opinion here

Reply:
I created the NumberProcessorResult to unify them.
The problem of ignoring is to import in the fetcher. I can just use number | undefined instead
I don't have a strong opinion here